cmd/migrate/migrate.go

```diff
@@ -144,5 +154,10 @@ func main() {
 	if visitedRes.RowsAffected > 0 {
 		logrus.Infof("Migrated %d user identity visited rows", visitedRes.RowsAffected)
 	}
+
+	if activeWorkspaceRes.RowsAffected > 0 {
+		logrus.Infof("Migrated %d user identity visited rows", activeWorkspaceRes.RowsAffected)
```

Comment:
FYI I have confirmed this works when migrating my DB locally (started chrome on main, create a user in the DB, and then checked out this PR). I see this from the migrate script:
```bash
2024/10/02 18:33:40 /Users/bflorkie/Development/github/RedHatInsights/chrome-service-backend/cmd/migrate/migrate.go:135
[0.442ms] [rows:1] UPDATE "user_identities" SET "active_workspace"='foobar',"updated_at"='2024-10-02 18:33:40.729' WHERE active_workspace IS NULL AND "user_identities"."deleted_at" IS NULL
INFO[0000] Migrated 1 user identity visited rows        
INFO[0000] Migration complete 
```

And then this from the rest endpoint after (note default workspace is "foobar"):
```bash
➜  chrome-service-backend git:(main) curl -H "x-rh-identity:<token>" http://localhost:8000/api/chrome-service/v1/user

{"data":{"createdAt":"0001-01-01T00:00:00Z","accountId":"12","firstLogin":true,  ... "uiPreviewSeen":false,"activeWorkspace":"foobar"}}
```